packages/terminal-extension/schema/plugin.json

```diff
@@ -59,6 +62,12 @@
       "description": "Whether to shut down or not the session when closing the terminal.",
       "type": "boolean",
       "default": false
+    },
+    "pasteWithCtrlV": {
```

Comment:
`pasteWithCtrlVNotMac`?

Reply:
As in Ctrl+V doesn't paste on a mac with this PR?  If it does, I don't see a reason to make a distinction.

Reply:
Actually, what we could do is set the default to false if on a mac and add a separate setting for `copyWithCtrlC` that is also false on a mac, I think that would be coherent.

Reply:
> As in Ctrl+V doesn't paste on a mac with this PR? If it does, I don't see a reason to make a distinction.

Standard browser behavior is that Ctrl+V doesn't paste on the mac.



> Actually, what we could do is set the default to false if on a mac and add a separate setting for `copyWithCtrlC` that is also false on a mac, I think that would be coherent.

Can we change defaults according to platform? I thought the json schema didn't allow changing defaults based on platform.

Reply:
Ah, shoot, I was thinking of the terminal options themselves, not the schema.   I think the setting should be ignored on a Mac and we just don't add the handler at all there.

Reply:
With documentation to that effect.

Reply:
Or we could be more explicit: `browserDefaultForCtrlVNotMac` or something. Instead of assuming that the browser will be pasting, just mentioning exactly what this option does, which is ignore ctrl+v in xterm.js.

Reply:
But that doesn't tell you anything about what Ctrl+C does.  That name is not read-able at all.

Reply:
Let me update the PR and see what you think.